api/src/main/resources/messages.properties

```diff
@@ -1,4 +1,8 @@
 ${project.parent.artifactId}.title=Bed Management Module
 ${project.parent.artifactId}.admissionLocations=Admission Locations
 ${project.parent.artifactId}.bedTypes=Bed Types
-${project.parent.artifactId}.bedTags=Bed Tags
\ No newline at end of file
+${project.parent.artifactId}.bedTags=Bed Tags
+bedPatientAssignment.endDatetime.beforeStartDatetime=Bed assignment's endDatetime cannot be before startDatetime
+bedPatientAssignment.endDatetime.afterVisitStopDatetime=Bed assignment's endDatetime cannot be after visit stopDatetime
+bedPatientAssignment.patient.alreadyAssigned=Cannot have multiple active bed assignments for the same patient
+bedPatientAssignment.visit.visitStoptimeBeforeEndDatetime=Visit stop time cannot be before bed assignment(s) end time
```

Comment:
Kindly I know its not that import though I believe ending the file with an empty to prevents unnecessary Git diff warnings.